test/test_transforms_v2.py

```diff
@@ -810,6 +826,30 @@ def test_max_size_error(self, size, make_input):
         with pytest.raises(ValueError, match=match):
             F.resize(make_input(self.INPUT_SIZE), size=size, max_size=max_size, antialias=True)
 
+    @pytest.mark.parametrize(
+        "max_size",
+        [min(list(INPUT_SIZE)) // 2, min(list(INPUT_SIZE)), max(list(INPUT_SIZE)), max(list(INPUT_SIZE)) * 2],
+    )
+    @pytest.mark.parametrize("device", cpu_and_cuda())
+    @pytest.mark.parametrize(
+        "make_input",
+        [
+            make_image_tensor,
+            make_image_pil,
+            make_image,
+            make_bounding_boxes,
+            make_segmentation_mask,
+            make_detection_masks,
+            make_video,
+        ],
+    )
+    def test_size_none(self, max_size, device, make_input):
+        check_transform(
```

Comment:
It seems that this test doesn't add much value over the existing `check_transform`, apart from testing different values of `max_size`? Maybe we can remove it and instead check for the core logic of `size=None, max_size=int` with some hard-coded values of `max_size` and `input_size`, checking the expected shape. Basically, something like this:

```py
    @pytest.mark.parametrize("input_size, max_size, expected_size",
        [
            ((10, 10), 10, (10, 10)),
            ((10, 20), 40, (20, 40)),
            ((20, 10), 40, (40, 20)),
            ((10, 20), 10, (5, 10)),
            ((20, 10), 10, (10, 5)),
        ]
    )
    @pytest.mark.parametrize("make_input", [make_image_tensor, make_image_pil])
    def test_resize_size_none(self, input_size, max_size, expected_size, make_input):
        img = make_input(input_size)
        out = F.resize(img, size=None, max_size=max_size)
        assert F.get_dimensions(out)[-2:] == list(expected_size)

```

Reply:
Sounds good!